pkgs/build-support/agda/default.nix

```diff
@@ -81,6 +81,9 @@ let
           runHook postInstall
         '';
 
+        # see https://github.com/NixOS/nixpkgs/issues/172752
+        LC_ALL = "C.UTF-8";
```

Comment:
This locale doesn't exist on darwin unfortunately, you may want to add a conditional to have it be `en_US.UTF-8` on that platform or something – if darwin support is imporant to you, that is.

(It's a real shame and major annoyance that there does not seem to be a _standard_ UTF-8 supporting locale on darwin, unless someone can enlighten me.)

Reply:
The `allPackages` build succeeds on x86_64-darwin, though: https://logs.nix.ci/?key=nixos/nixpkgs.213896&attempt_id=b2775f53-d698-4681-85f3-0d1ecd918022

Reply:
Okay, then the linked issue doesn't affect darwin apparently, as the locale indeed doesn't work:

```
sh: warning: setlocale: LC_ALL: cannot change locale (C.UTF-8): No such file or directory
```

Maybe we can just not set it for darwin then…

Reply:
Okay, will try that!